SCClassLibrary/External/Ableton/LinkClock.sc

```diff
@@ -0,0 +1,77 @@
+// clock with Ableton Link synchronization
+
+LinkClock : TempoClock {
+
+	var <>tempoChanged;
+
+	*newFromTempoClock { arg clock;
```

Comment:
it's part of our [style guidelines](https://github.com/supercollider/supercollider/wiki/%5BWIP%5D-Code-style-guidelines#rule-use-pipes-instead-of-the-arg-keyword-to-express-parameter-lists) to use pipes instead of arg, can you please change that in this file?